src/OrbitQt/CallTreeWidget.h

```diff
@@ -69,6 +71,7 @@ class CallTreeWidget : public QWidget {
   void OnSearchLineEditTextEdited(const QString& text);
   void OnSearchTypingFinishedTimerTimout();
   void OnSliderValueChanged(int value);
+  void OnLeaveButtonClicked();
```

Comment:
I suggest `OnLeaveInspectionButtonClicked`, "leave" alone is too generic.

Reply:
Added.